doc/content/source/bcs/EconomouDielectricBC.md

```diff
@@ -1,20 +1,33 @@
 # EconomouDielectricBC
 
-!alert construction title=Undocumented Class
-The EconomouDielectricBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/EconomouDielectricBC
 
 ## Overview
 
-!! Replace these lines with information regarding the EconomouDielectricBC object.
+`EconomouDielectricBC` is an type of [`PenaltyDirichletBC`](/bcs/ADPenaltyDirichletBC.md) for the potential on the boundary of a grounded ideal dielectric.
+
+The potential at the boundary of a grounded ideal dielectric is defined as
+
+\begin{equation}
+\frac{\epsilon_{i}}{d_{i}}\frac{\partial V_{i}}{\partial t} = e(\Gamma_{+} \cdot \textbf{n} -\Gamma_{e} \cdot \textbf{n})+\epsilon_{0}\frac{\partial (E \cdot \textbf{n}) }{\partial t} \\[10pt]
+E = \text{-} \nabla (V)\\[10pt]
+\Gamma_{e} \cdot \textbf{n}  = \frac{1}{4}\sqrt{\frac{8 k T_{e}}{\pi m_{e}}} \ n_e - \gamma \Gamma_{+} \cdot \textbf{n} \\[10pt]
+\Gamma_{+} \cdot \textbf{n}  = a \ \mu_{+} \ \text{-} \nabla (V) \cdot \textbf{n} \ n_{+} \\[10pt]
+a =
+\begin{cases}
+1, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} > 0\\
+0, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \leq 0\\
+\end{cases}
+\end{equation}
+
+Where $\epsilon_{i}$ is the permittivity of the dielectric, $d_{i}$ is the thickness of the dielectric, $V_{i}$ is the voltage on the dielectric, $\textbf{n}$ is the normal to the boundary, $e$ is the elemental charge, $\epsilon_{0}$ is the permittivity of free space, and $E$ is the E-field normal to the dielectric. $\Gamma_{e}$ and $\Gamma_{+}$ are the electron and ion outflow flux and are defined with the [`SakiyamaElectronDiffusionBC`](/bcs/SakiyamaElectronDiffusionBC.md), [`SakiyamaSecondaryElectronBC`](/bcs/SakiyamaSecondaryElectronBC.md) and [`SakiyamaIonAdvectionBC`](/bcs/SakiyamaIonAdvectionBC.md) (please refer to those BC's for more information on the fluxes).
 
 ## Example Input File Syntax
 
-!! Describe and include an example of how to use the EconomouDielectricBC object.
+An example of how to use `EconomouDielectricBC` can be found in the
+test file `2D_RF_Plasma_actions.i`.
```

Comment:
@gsgall I am finally addressing this PR before pushing the microwave coupling PR. For this comment, I don't have a problems with removing this phrasing, but if we do removing it, it should also be removed from all auxkernel and kernel documentation. That can either be about of this PR and the microwave PR, since most of the documentation will need to be edited for the coupling between either electrostatic or electromagnetic (I personal think it should be the next PR, since this one is just for BC).